<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1025</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QListWidget" name="musicList">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>30</y>
     <width>451</width>
     <height>301</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="widget_4" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>460</y>
     <width>971</width>
     <height>121</height>
    </rect>
   </property>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>293</width>
      <height>86</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="prevBtn">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="playerBtn">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="nextBtn">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="modeBtn">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>9</y>
      <width>461</width>
      <height>101</height>
     </rect>
    </property>
    <widget class="QLabel" name="albumPic">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>60</width>
       <height>60</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>60</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QSlider" name="processBar">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>60</y>
       <width>221</width>
       <height>22</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="songName">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>151</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei UI Light</family>
       <pointsize>8</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>我怀念的 - 孙燕姿</string>
     </property>
    </widget>
    <widget class="QLabel" name="currentTime">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>50</y>
       <width>61</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>03:51/</string>
     </property>
    </widget>
    <widget class="QLabel" name="totalTime">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>50</y>
       <width>60</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>04:50</string>
     </property>
    </widget>
    <widget class="QLabel" name="currentLyric">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>12</y>
       <width>231</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>生日快乐</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_3" native="true">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>20</y>
      <width>181</width>
      <height>80</height>
     </rect>
    </property>
    <widget class="QSlider" name="soundBar">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>35</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="soundBtn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
